### DISCUSSION TITLE
参考文献引用不成功

### DISCUSSION THREAD

**Discussion by Yuly143982** [sikouhjw/gdutthesis#58] [Q&A] [answered]
下的最新版的，然后用winEdt+texlive编写，把原来bib里面的参考文献全删了，自己新加进去了，没改文件名。
引用参考文献的时候，上标一直是问号，也试着编译了四五遍，都一样。后面也没出现参考文献的目录，结论之后直接是成果。
引用的时候能弹出来存在那个参考文献，就是编译后没有。
![1648280866(1)](https://user-images.githubusercontent.com/72240527/160230123-38e326e3-b2d1-4697-938e-a5ab497229cf.png)
![1648280892(1)](https://user-images.githubusercontent.com/72240527/160230133-6a56bff2-8f45-4961-8166-e469d61ab428.png)
![1648280910(1)](https://user-images.githubusercontent.com/72240527/160230145-18dcc397-d650-4358-b0f3-7e6a4844c5be.png)
![1648280966(1)](https://user-images.githubusercontent.com/72240527/160230170-4f61c909-9b47-4e57-af96-352e95afb0fe.png)

**Comment by Yuly143982** [comment]
用在线版的没这个问题，不知道我哪里 搞错了

**Comment by sikouhjw** [comment, marked as answer]
问题在于你没有进行 `bibtex` 的操作，建议你
1. 命令行输入 `texdoc install-latex`，阅读如何命令行编译；
2. 命令行编译 `xelatex main`->`bibtex main`->`xelatex main`->`xelatex main`，此时可以发现『问号』变成了引用的数字；
3. 再将辅助文件删除，尝试用 `latexmk -xelatex main` 去一键编译；
4. 此时你知道了『编辑器』编译的本质，可以参考《WinEdit 和 Textstudio 分别如何使用 Bibtex 编译参考文献》[^1]《胡老师教你：Winedt 一键编译 LaTeX 文档》[^2]去用 Winedt 编译了；
5. 如果用 Winedt 仍然编译不了，建议使用命令行编译、TeXworks（texlive 自带）或者 Overleaf 来编译。

[^1]: https://www.cnblogs.com/pupilLZT/p/13162698.html
[^2]: https://ask.latexstudio.net/ask/article/103.html

**Reply by Yuly143982** [reply]
哦哦，谢谢谢谢，懂了，一开始看那个 教学视频，我以为是编译四次xelatex main呢，谢谢谢谢
